types/selenium-webdriver/lib/error.d.ts

```diff
@@ -0,0 +1,344 @@
+/**
+ * Enum of legacy error codes.
+ * TODO: remove this when all code paths have been switched to the new error
+ * types.
+ * @deprecated
+ * @enum {number}
+ */
+export enum ErrorCode {
```

Comment:
This should be a `const` with an object literal type instead of an `enum`. TypeScript enums have reverse keys (`ErrorCode.Success === 0` and `ErrorCode["0"] === "Success"`) whereas the JS only has forward keys.

Reply:
Ooh good to know